operator/node.go

```diff
@@ -149,7 +149,11 @@ func (n *operatorNode) Start(logger *zap.Logger) error {
 	}()
 
 	go n.ticker.Start(logger)
-	go n.listenForCurrentSlot(logger)
+	go func() {
+		if err := n.listenForCurrentSlot(logger); err != nil {
+			logger.Fatal(err.Error())
```

Comment:
```suggestion
			logger.Fatal("failed listening for current slot", zap.Error(err))
```

wdyt?

Reply:
I didn't do that to keep the same error, but after looking at it again I think logs would look good with your suggestion. I agree